### PR TITLE
Verifier cleanup/bugfix golem node tests fails

### PR DESCRIPTION
**Pull request by kbeker**
Bugfix for problems which occurred during testing Golem with node tests

### CODE REVIEW

**Review by Jakub89 — APPROVED** [review]
tenor

**Review by dybi — APPROVED** [review]
Just please make sure all tests are passing ;)

van_dame_approves

File: apps/core/task/coretask.py [hunk 1]
```diff
@@ -205,6 +205,8 @@ def computation_finished(self, subtask_id, task_result,
             logger.info("Not accepting results for %s", subtask_id)
             return
         self.subtasks_given[subtask_id]['status'] = SubtaskStatus.verifying
+        self.subtasks_given[subtask_id]['entrypoint'] = \
+            "python3 /golem/entrypoints/verifier_entrypoint.py"
```
Comment (Krigpl) [inline apps/core/task/coretask.py:209]: Why is this here? This is Blender specific and this is a `CoreTask` that shouldn't know anything about that. Please remove that.
